README.md

```diff
@@ -8,11 +8,12 @@ The Stan Math Library is licensed under the [new BSD license](https://raw.github
 
 Required Libraries
 ------------------
-Stan Math depends on three libraries:
+Stan Math depends on four libraries:
```

Comment:
Given the links are the same, I think "CVODES" should be replaced with "Sundials".  But I don't understand why the versions would be different.  If they're really different libraries then we can list them separately, but we're including all sorts of things from Eigen and Boost together.

Reply:
Somehow IDAS is of a different version with other SUNDIALS libs. Here I'm using the latest IDAS, which is 2.1.0. The latest CVODES is 3.1.0, while the one in current stan is 2.9.0. 

CVODES changed some of its APIs in 3.0.0, so the current bdf solver implementation would break in the latest CVODES(APIs like `CVDense` used in stan have been removed).